stdlib/asm/word.masm

```diff
@@ -0,0 +1,228 @@
+#! Reverses order of the first four elements on the stack
+#!
+#! Inputs:  [a, b, c, d, ...]
+#! Outputs: [d, c, b, a, ...]
+#!
+#! Cycles: 3
+export.reverse
+	movdn.3
+	swap.2
+end
+
+#! Returns a boolean indicating whether the input word is [0, 0, 0, 0].
+#!
+#! Inputs:  [INPUT_WORD]
+#! Outputs: [is_word_zero, INPUT_WORD]
+#!
+#! Where:
+#! - INPUT_WORD is the word to compare against [0, 0, 0, 0].
+#! - is_empty_word is a boolean indicating whether INPUT_WORD is all zeros.
```

Comment:
I would use the `is_empty_word` instead of the `is_word_zero`, but it's not a strong preference. But the main idea is to make the name in `Outputs` and in the description the same.